#!/usr/bin/python3
class raise_exception(Exception):
    pass
